lang/src/main/java/mumbler/truffle/node/builtin/lang/ReadBuiltinNode.java

```diff
@@ -20,7 +20,7 @@
     @Specialization
     public Syntax read(VirtualFrame virtualFrame, String str) {
         try {
-        	Source source = Source.fromText(str, "<read>");
+        	Source source = Source.newBuilder("Mumbler", str, "<read>").build();
```

Comment:
here "Mumbler" is upper-case. Is that intentional? If not, use the same constant from MumberLanguage as suggested.

Reply:
SHould be lower case, will use the constant